vignettes/Introduction.Rmd

```diff
@@ -254,7 +254,7 @@ cq$get_total()
 
 ### Fields containing reference objects
 
-If your R6 class contains any fields that also have reference semantics (e.g., other R6 objects, and environments), those fields should be populated in the `initialize` method. If the field set to the reference object directly in the class definition, that object will be shared across all instances of the R6 objects. Here's an example:
+If your R6 class contains any fields that also have reference semantics (e.g., other R6 objects, environments, etc.), those fields should be populated in the `initialize` method. If the field is directly set to the reference object in the class definition, that object will be shared across all instances of the R6 objects. Here's an example:
```

Comment:
The "etc" is not needed because "e.g." means "for example"; it already implies that the following items are not exhaustive.

For the second change, this version is slightly more accurate:

```suggestion
If your R6 class contains any fields that also have reference semantics (e.g., other R6 objects, and environments), those fields should be populated in the `initialize` method. If the field is set to the reference object directly in the class definition, that object will be shared across all instances of the R6 objects. Here's an example:
```

Reply:
Included both of these changes. Thanks.